test/acceptance/tests.py

```diff
@@ -15,8 +16,10 @@
     SubmissionPage, AssessmentPage, GradePage, StaffAreaPage
 )
 
+# This value is generally used in jenkins, but not locally
+PROFILING_ENABLED = os.environ.get('ORA_PROFILING_ENABLED', False)
 
-def retry(tries=4, delay=3, backoff=2):
+def retry(tries=2, delay=4, backoff=2):
```

Comment:
Some our biggest tests are more than 3-4 minutes on jenkins, which means trying them more than 2x leads to hitting the SauceLabs imposed 10-minute timeout failure. I think having 2 tries as a default is a good balance between letting one sporadic failure crash a test run, and holding everything up in the case of legitimate errors.